File: NetCore/CascadingSelectBoxesSample/Models/Address.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CascadingSelectBoxesSample.Models
{
    public class Address
    {
        public int? StateID { get; set; }
        public int? CityID { get; set; }

        public static readonly Address Empty = new Address();
    }
}

File: VB/NetCore/CascadingSelectBoxesSample/Models/Address.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Threading.Tasks

Namespace CascadingSelectBoxesSample.Models
	Public Class Address
		Public Property StateID() As Integer?
		Public Property CityID() As Integer?

		Public Shared ReadOnly Empty As New Address()
	End Class
End Namespace
